src/clj_watson/logic/dependency_check/vulnerability.clj

```diff
@@ -4,17 +4,23 @@
   (:import
    (org.owasp.dependencycheck.dependency Vulnerability)))
 
-(defn ^:private cvssv3 [vulnerability]
-  (try
-    (some-> vulnerability .getCvssV3 .getCvssData .getBaseScore)
-    (catch Exception _
-      nil)))
-
-(defn ^:private severity [vulnerability]
-  (try
-    (some-> vulnerability .getCvssV3 .getCvssData .getBaseSeverity str)
-    (catch Exception _
-      nil)))
+(defn ^:private base-score [cvss]
```

Comment:
In the old code, each of these chains of calls is wrapped in `try` / `catch` -- is there any danger that the call chains can actually throw or was the old code overly defensive?

Reply:
My feeling was the old code was hiding potential issues. 
I don't see why we should normally encounter an exception here (and didn't see a rationale in the git history).
If we learn otherwise, we can adapt and address.

Reply:
Fair enough. I wasn't sure whether those methods were declared to throw exceptions.